proto/scheduler.proto

```diff
@@ -70,11 +72,8 @@ message TaskSize {
 
 message ScheduleTaskRequest {
   string task_id = 1;
-  TaskSize task_size = 2;
-  bytes serialized_task = 3;
-  string os = 4;
-  string arch = 5;
-  string pool = 6;
+  reserved 2 to 6;
+  build.bazel.remote.execution.v2.ExecutionTask task = 7;
```

Comment:
The reason this was a serialized proto before was that we added the task queueing system onto a pre-existing remote execution system behind a load balancer. The Task queue was designed to be "remote exec" agnostic -- at first.

(This is the inspiration and rough guide for the scheduling system btw: https://cs.stanford.edu/~matei/papers/2013/sosp_sparrow.pdf)

Anyway, I'm OK if you merge these -- but just want to make doubly sure this is what you intend to do and that you think that makes sense for the future.

Reply:
Thanks for the feedback.

Upon further thought I think it makes sense to keep this opaque. 
There's potential that other use cases might come up where we may want to reuse the scheduling infra.